api/specs.proto

```diff
@@ -255,3 +255,21 @@ message ClusterSpec {
 	// Existing tasks will continue to use the previous default until rescheduled.
 	Driver default_log_driver = 7;
 }
+
+// NetworkAttachmentSpec specifies the allocation information
+// for the network attachment.
+message NetworkAttachmentSpec {
```

Comment:
This should actually be `NetworkAttachmentConfig`. `Spec` should only _ever_ be user input.